platform/drm/cog-platform-drm.c

```diff
@@ -214,119 +204,83 @@ static struct {
     struct wpe_view_backend *backend;
 } wpe_view_data;
 
-
 static void
-init_config (CogShell *shell)
+init_config(CogDrmPlatform *self, CogShell *shell, const char *params_string)
 {
     drm_data.device_scale = cog_shell_get_device_scale_factor (shell);
     g_debug ("init_config: overriding device_scale value, using %.2f from shell",
              drm_data.device_scale);
 
     GKeyFile *key_file = cog_shell_get_config_file (shell);
-    if (!key_file)
-        return;
 
-    {
-        g_autoptr(GError) lookup_error = NULL;
-        gboolean value = g_key_file_get_boolean (key_file,
-                                                 "drm", "disable-atomic-modesetting",
-                                                 &lookup_error);
-        if (!lookup_error) {
-            drm_data.atomic_modesetting = !value;
-            g_debug ("init_config: atomic modesetting reconfigured to value '%s'",
-                     drm_data.atomic_modesetting ? "true" : "false");
+    if (key_file) {
+        {
+            g_autoptr(GError) lookup_error = NULL;
+            gboolean value = g_key_file_get_boolean(key_file, "drm", "disable-atomic-modesetting", &lookup_error);
+            if (!lookup_error) {
+                drm_data.atomic_modesetting = !value;
+                g_debug("init_config: atomic modesetting reconfigured to value '%s'",
+                        drm_data.atomic_modesetting ? "true" : "false");
+            }
         }
-    }
 
-    {
-        g_autoptr(GError) lookup_error = NULL;
-        gdouble value = g_key_file_get_double (key_file,
-                                               "drm", "device-scale-factor",
-                                               &lookup_error);
-        if (!lookup_error) {
-            drm_data.device_scale = value;
-            g_debug ("init_config:overriding device_scale value, using %.2f from config",
-                     drm_data.device_scale);
+        {
+            g_autoptr(GError) lookup_error = NULL;
+            gdouble           value = g_key_file_get_double(key_file, "drm", "device-scale-factor", &lookup_error);
```

Comment:
This seems like weird spacing to me, is this part of the style...?

Reply:
Yes, this is `clang-format` being funny about aligning variable names; the `g_autoptr()` macro makes it look even weirder than it would normally be. I am not 100% convinced about that formatting rule. Even funnier: there is a similar instance right above which is in theory badly formatted, and `clang-format` does not complain about it 🙃

Reply:
Oh well, I went ahead and tried locally disabling the `AlignConsecutiveDeclarations` option locally and it indeed makes this ugly instance go away, but at the cost of making other parts of the code not as neat and readable. I'll workaround adding an empty line between the `GError` declaration and the next line, to avoid the formatter trying to align them in this particular case.